test-mocks/endpoint-mock/src/main/java/org/divy/common/bo/endpoint/test/InMemoryBOManager.java

```diff
@@ -33,15 +33,13 @@ public E delete(E businessObject) {
 
     @Override
     public List<E> list() {
-        ArrayList<E> arrayList = new ArrayList<>();
-        arrayList.addAll(inMemoryMap.values());
+        ArrayList<E> arrayList = new ArrayList<>(inMemoryMap.values());
         return arrayList;
     }
 
     @Override
     public List<E> search(Query businessObjectQuery) {
-        ArrayList<E> arrayList = new ArrayList<>();
-        arrayList.addAll(inMemoryMap.values());
+        ArrayList<E> arrayList = new ArrayList<>(inMemoryMap.values());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "arrayList". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)